### PR TITLE
Feature/improve release

### PR DESCRIPTION
**Pull request by white-gecko**

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    desc: Build the whl package
+    cmds:
+      - poetry build
+
+  build:docker:
+    desc: Build the docker image
+    deps:
+      - task: build
+    cmds:
+      - docker build --build-context dist=./dist -t aksw/quitstore .
```
Comment (splattater) [inline Taskfile.yml]: Throws an error "unknown flag: --build-context" but package and whl were built in "./dist"

Comment (white-gecko) [reply Taskfile.yml]: fixed

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
              poetry run pip install pygit2
              poetry install --no-root
       - name: Check Conding style
-        run: poetry run pylava
+        run: poetry run pylama -i E402,C
```
[the remark this answers is not in the document]
Comment (white-gecko) [reply .github/workflows/build.yml:35]: yes, cf: https://github.com/AKSW/QuitStore/pull/305/files#diff-a978c051d09cb6a07b836a846f4e6ca9e5829ab6bd1c346fb34d2ca726a6adeb

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -47,18 +47,47 @@ jobs:
...
+
   release:
     runs-on: ${{ matrix.os }}
-    needs: test
+    needs: build
     strategy:
       matrix:
         python-version: [3.9]
-        os: ['windows-2019', 'ubuntu-18.04', 'macos-11']
+        os: ['windows-2019', 'ubuntu-latest', 'macos-11']
```
[the remark this answers is not in the document]
Comment (white-gecko) [reply .github/workflows/build.yml]: fixed

**Review by splattater — APPROVED** [review]

**Pull request merged**